app/src/main/java/com/oguzdogdu/walliescompose/features/search/SearchScreen.kt

```diff
@@ -131,6 +131,15 @@ fun SearchScreenRoute(
         viewModel.handleUIEvent(SearchEvent.GetAppLanguageValue)
     }
 
+    LaunchedEffect(key1 = Unit) {
```

Comment:
Yine viewmodelde ki applanguage değerini alıp buraya paslayabilirsin ancak LaunchedEffect onCreate ile birlikte çalışacağı için     
```
LifecycleResumeEffect(key1 = queryFromDetail) {
        if (queryFromDetail.isNullOrBlank().not()) {
            viewModel.handleUIEvent(
                SearchEvent.EnteredSearchQuery(
                    queryFromDetail.orEmpty(),
                    appLanguage
                )
            )
        }
        onPauseOrDispose {
            viewModel.handleUIEvent(SearchEvent.EnteredSearchQuery(query = "", appLanguage))
        }
    }
``` 
şeklinde handle etmen daha doğru olabilir.

Reply:
appLanguage verince ilginç bir şekilde çalışmıyor, sanırım end-point ile alakalı, dediğin gibi yaptım ama olmadı null verince düzeldi